src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Create `.env` file and move BASE_URL there, so you'll be able to use it everywhere you need, instead of redeclaring it